vendor-extra/JatsContentBundle/tests/EventListener/TitleListenerTest.php

```diff
@@ -4,15 +4,52 @@
 
 namespace tests\Libero\JatsContentBundle\EventListener;
 
-use Libero\ContentPageBundle\Event\CreateContentPageEvent;
 use Libero\JatsContentBundle\EventListener\TitleListener;
+use Libero\LiberoPageBundle\Event\CreatePageEvent;
 use PHPUnit\Framework\TestCase;
-use tests\Libero\ContentPageBundle\XmlTestCase;
+use tests\Libero\LiberoPageBundle\PageTestCase;
+use tests\Libero\LiberoPageBundle\XmlTestCase;
 
 final class TitleListenerTest extends TestCase
 {
+    use PageTestCase;
     use XmlTestCase;
 
+    /**
+     * @test
+     */
+    public function it_does_nothing_if_it_is_not_a_content_page() : void
+    {
+        $listener = new TitleListener();
+
+        $document = $this->loadDocument(
+            <<<XML
+<?xml version="1.0" encoding="UTF-8"?>
+<libero:item xmlns:libero="http://libero.pub" xmlns:jats="http://jats.nlm.nih.gov">
```

Comment:
do we really need all this fixture data if the test is for `it_does_nothing_if_...`?

Reply:
Was trying to avoid triggering other reasons to bail out (so everything's valid apart from the thing that's being tested). Does make these verbose though.